docs/appendix/server-component-manifest.md

```diff
@@ -27,6 +27,14 @@ The manifest file contains the following properties:
 - **`license`**
 
   (Optional but recommended) Defines the license code of the component. For example, Zowe core components have `EPL-2.0` value in this field.
+  
+- **`schemas`**
+
+  Defines the location of json schema files that are compatible with certain portions of Zowe as denoted by each child property.
+  
+  * **`config`**
```

Comment:
Is that ok to make it plural so it aligns the `configs` section in manifest.yaml. My understanding this is the schema to describe `configs` sections. Not sure if I understand correctly.

Reply:
ok makes sense.